sale_date_order_editable/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 C2i Change 2 improve <soporte@c2i.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale date order editable",
+    "version": "15.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Uncategorized",
```

Comment:
Dear @emagdalenaC2i , could you please add the categoty please? this is really useful when searching in odoo-community.org/shop, and other places and I think easy to solve :-)

Example: https://github.com/odoo/odoo/blob/15.0/addons/sale_margin/__manifest__.py#L7

Reply:
done